docs/user/resources/02-logpipeline.md

```diff
@@ -199,23 +199,21 @@ The status of the LogPipeline is determined by the condition types `AgentHealthy
+| ---------------------- | ---------------- | --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AgentHealthy           | True             | AgentReady                  | Fluent Bit agent DaemonSet is ready                                                                                                                                                                                                 |
+| AgentHealthy           | False            | AgentNotReady               | Fluent Bit agent DaemonSet is not ready                                                                                                                                                                                             |
+| ConfigurationGenerated | True             | ConfigurationGenerated      |                                                                                                                                                                                                                                     |
+| ConfigurationGenerated | False            | ReferencedSecretMissing     | One or more referenced Secrets are missing                                                                                                                                                                                          |
+| ConfigurationGenerated | False            | UnsupportedLokiOutput       | grafana-loki output is not supported anymore. For integration with a custom Loki installation, use the `custom` output and follow [Intergrate with Loki](https://kyma-project.io/#/telemetry-manager/user/integration/loki/README). |
+| ConfigurationGenerated | False            | TLSCertificateInvalid       | TLS certificate invalid                                                                                                                                                                                                             |
+| ConfigurationGenerated | False            | TLSCertificateExpired       | TLS certificate expired on YYYY-MM-DD                                                                                                                                                                                               |
+| ConfigurationGenerated | True             | TLSCertificateAboutToExpire | TLS certificate is about to expire, configured certificate is valid until YYYY-MM-DD                                                                                                                                                |
```

Comment:
Those tables are randomly sorted, which makes them harder to read. I suggest we sort them:
1. By the status (True, then False)
2. If the status is the same, then alphabetically

Reply:
I also mean the other md files